src/wallet/tx_builder.rs

```diff
@@ -303,6 +315,14 @@ impl<D: Database, Cs: CoinSelectionAlgorithm<D>> TxBuilder<D, Cs> {
             phantom: PhantomData,
         }
     }
+
+    /// Returns true if an absolute fee was specified
+    pub fn has_absolute_fee(&self) -> bool {
```

Comment:
This method can probably also be removed if you implement the changes I proposed in `src/mod/wallet.rs`.